app/templates/check_running.html

```diff
@@ -0,0 +1,19 @@
+{% extends "custom_layout.html" %}
+
+{% block header_additional %}
+    <meta http-equiv="refresh" content="5">
+{% endblock %}
+
+{% block body %}
+    <div class="container">
+        <div class="card">
+            <div class="card-body">
+                <h5 class="card-title">{% trans %}Domain analysis is running{% endtrans %}</h5>
+                <div class="card-text">
+                    <img class="spinner waiting" src="/static/images/spinner.svg" />
+                    <h5 class="waiting">{% trans %}Waiting for the domain analysis to finfish{% endtrans %}</h5>
+                </div>
+            </div>
+        </div>
+   </div>
+{% endblock %}
```

Comment:
IMO it's not the most beautiful UI possibl:

![image](https://github.com/CERT-Polska/mailgoose/assets/1233067/6360ebbb-fb78-4ee9-86ff-8778200cf541)

Reply:
it's still - need some help in ui design?